packages/node-red-s1seven-api/lib/tokens/get-token.html

```diff
@@ -11,8 +11,8 @@
       apiConfig: { value: '', type: 'api-config' },
     },
     inputs: 1,
-    outputs: 2,
-    outputLabels: OUTPUT_LABELS,
+    outputs: 3,
+    outputLabels: [...OUTPUT_LABELS, 'event'],
```

Comment:
since all nodes will be able to produce this event, you should already merge `event` with `OUTPUT_LABELS`